src/main.cpp

```diff
@@ -2176,6 +2176,14 @@ bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockIndex* pin
         flags |= SCRIPT_VERIFY_CHECKLOCKTIMEVERIFY;
     }
 
+    // Start enforcing BIP68 (sequence locks) and CHECKSEQUENCEVERIFY (BIP112) for block.nVersion=5
+    // blocks, when 75% of the network has upgraded:
+    int nLockTimeFlags = 0;
+    if (block.nVersion >= 5 && IsSuperMajority(5, pindex->pprev, chainparams.GetConsensus().nMajorityEnforceBlockUpgrade, chainparams.GetConsensus())) {
```

Comment:
So what is your proposed fix? Or do you want to risk forking prematurely? You do realize this could put Core on a minority chain in the event that Classic has 50+ % hashing power?